### DISCUSSION TITLE
Electron-forge issue: running as root without --no-sandbox is not supported

### DISCUSSION THREAD

**Discussion by randyb100** [ohmplatform/FreedomGPT#106] [Q&A]
Hi all,

I'm having an issue getting this to work. I tried editing the package.json file to add in the --no-sandbox argument, but it only works if I change it to say "yarn --no-sandbox." Then it doesn't load the package at all.

This is on Linux if that matters any. I am not using the Docker version. 

It does link me to https://crbug.com/638180 and appears to be a Chromium issue. I simply don't know where to add in the --no-sandbox argument in the code! Can anyone help?

**Comment by Mathboss42** [comment]
Hi, late to the party, so I'm pretty sure you've figured something out since, but leaving this reply here in case it helps somebody else.

I am running an electron project with electron-forge inside a node:22 Docker container on WSL2.

I've been having issues with the same argument while trying to run the `electron-forge start` command (through `npm run start`). I tried a bunch of things such as adding the `-- --no-sandbox` arguments directly to the command in the `package.json` or in the command line, or using `app.commandLine.appendSwitch('--no-sandbox')`, or adding the argument to the `startElectronArgs` array in `forge.config`, but to no avail, I kept getting the same "running as root without --no-sandbox is not supported" error on startup.

The thing that ended up working for me was setting the environment variable `DISABLE_ELECTRON_SANDBOX` to `1`. Not exactly sure yet why the other methods didn't work, but thought I would leave this here for someone else to find in case it helped.

Have a good one.
